.github/workflows/ci.yaml

```diff
@@ -30,85 +30,69 @@ jobs:
         mc-rtc-version: [head, stable]
         motion-type: [PushCartWaypoint]
         exclude:
           - build-type: Debug
             mc-rtc-version: stable
+          # Some packages have not been released onto the stable mirror yet
+          - catkin-build: standalone
+            mc-rtc-version: stable
         include:
           - os: ubuntu-20.04
             build-type: RelWithDebInfo
             mc-rtc-version: head
+            catkin-build: catkin
+            motion-type: PushCartVelMode
+          - os: ubuntu-20.04
+            build-type: RelWithDebInfo
+            mc-rtc-version: head
+            catkin-build: standalone
             motion-type: PushCartVelMode
     runs-on: ${{ matrix.os }}
     steps:
-      - name: Set ROS version
-        run: |
-          if [ "${{ matrix.os }}" == "ubuntu-20.04" ]; then
-            echo "ROS_DISTRO=noetic" >> $GITHUB_ENV
-            echo "PYTHON_PACKAGE_PREFIX=python3" >> $GITHUB_ENV
-          else
-            echo "ROS_DISTRO=melodic" >> $GITHUB_ENV
-            echo "PYTHON_PACKAGE_PREFIX=python" >> $GITHUB_ENV
-          fi
-      - name: Install ROS
-        run: |
-          set -e
-          set -x
-          sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
-          wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
-          sudo apt-get update -y -qq
-          sudo apt-get install -y -qq ros-${ROS_DISTRO}-ros-base ${PYTHON_PACKAGE_PREFIX}-catkin-tools ${PYTHON_PACKAGE_PREFIX}-rosdep ${PYTHON_PACKAGE_PREFIX}-wstool doxygen graphviz
-      - name: Install mc_rtc
-        run: |
-          set -e
-          set -x
-          curl -1sLf 'https://dl.cloudsmith.io/public/mc-rtc/${{ matrix.mc-rtc-version }}/setup.deb.sh' | sudo -E bash
-          sudo apt install -y -qq libmc-rtc-dev mc-rtc-utils mc-state-observation jvrc-choreonoid libcnoid-dev ros-${ROS_DISTRO}-mc-rtc-plugin libeigen-qld-dev
+      - name: Install dependencies
+        uses: jrl-umi3218/github-actions/install-dependencies@master
+        with:
+          ubuntu: |
+            apt-mirrors:
+              mc-rtc:
+                cloudsmith: mc-rtc/${{ matrix.mc-rtc-version }}
+            apt: libmc-rtc-dev mc-rtc-utils mc-state-observation doxygen graphviz jvrc-choreonoid choreonoid libcnoid-dev
+          ros: |
+            apt: ros-base mc-rtc-plugin eigen-conversions
+      - name: Install standalone dependencies
+        if: matrix.catkin-build == 'standalone'
+        uses: jrl-umi3218/github-actions/install-dependencies@master
+        with:
+          compiler: ${{ matrix.compiler }}
+          build-type: ${{ matrix.build-type }}
+          ros: |
+            apt: sbpl
+          ubuntu: |
+            apt: libforcecontrolcollection-dev libtrajectorycollection-dev libcentroidalcontrolcollection-dev libmc-rtc-dev
+          github: |
+            - path: isri-aist/BaselineFootstepPlanner
```

Comment:
```suggestion
```
`BaselineFootstepPlanner` is optional dependency for `BaselineWalkingController` and it can be skipped for this CI.